HousingSearchApi.Tests/V1/Helper/TenureListSortFactoryTests.cs

```diff
@@ -0,0 +1,49 @@
+using FluentAssertions;
+using Hackney.Shared.HousingSearch.Gateways.Models.Tenures;
+using HousingSearchApi.V1.Boundary.Requests;
+using HousingSearchApi.V1.Infrastructure.Sorting;
+using Xunit;
+
+namespace HousingSearchApi.Tests.V1.Helper
+{
+    public class TenureListSortFactoryTests
+    {
+        private readonly SortFactory _sut;
+        private static readonly string _tenureStartDateFieldName = "tenureStartDate";
+
+        public TenureListSortFactoryTests()
+        {
+            _sut = new SortFactory();
+        }
+
+        [Fact]
+        public void ShouldNotSortByDefault()
+        {
+            // Arrange + Act
+            var result = _sut.Create<QueryableTenure, GetTenureListRequest>(new GetTenureListRequest());
+
+            // Assert
+            result.Should().BeOfType(typeof(DefaultSort<QueryableTenure>));
+        }
```

Comment:
Yes, that way all paths return implementation of ISort<T>, but the default one doesn't of course sort by anything. I guess this approach makes sense just in case you ever wanted to apply some default sorting.

Reply:
As for the `QueryableTenureHelper` setup, I started with the generic one like you mentioned, but then I was thinking ahead about introducing more entity types in the mix and decided the files would get unnecessarily long over time. Personally I prefer split files over regions, so that's my thinking behing that structure.